src/elvis_style.erl

```diff
@@ -394,14 +394,27 @@ dont_repeat_yourself(Config, Target, RuleConfig) ->
 max_module_length(Config, Target, RuleConfig) ->
     MaxLength = maps:get(max_length, RuleConfig, 500),
     IgnoreModules = maps:get(ignore, RuleConfig, []),
+    CountComments = maps:get(count_comments, RuleConfig, false),
```

Comment:
can we make it work the other way around? like `ignore_comments` instead of `count_comments`?